documentation/docs/src/nodes/mix-node-setup.md

```diff
@@ -228,12 +228,14 @@ sudo ufw enable
 sudo ufw status
 ```
 
-Finally open your mix node's p2p port, as well as ports for ssh, http, and https connections, and ports `8000` and `1790` for verloc and measurement pings:
+Finally open your mix node's p2p port, as well as ports for ssh and ports `8000` and `1790` for verloc and measurement pings:
 
 ```
-sudo ufw allow 1789,1790,8000,22,80,443/tcp
+sudo ufw allow 1789,1790,8000,22/tcp
 # check the status of the firewall
 sudo ufw status
+# restart firewall
+sudo sytstemctl restart ufw
```

Comment:
This is not needed

Reply:
agree @Pawnflake can you please remove these lines for me? thanks